.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.10
```

Comment:
Added this because Python 3.9 does not work with `click==8.2.1`, and several package managers support `.tool-versions` files.